addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -2285,6 +2285,9 @@ QUnit.test('new message separator is shown in a chat window of a chat on receivi
         '.o_MessageList_separatorNewMessages',
         "should display 'new messages' separator in the conversation, from reception of new messages"
     );
+    await afterNextRender(() =>
+        document.querySelector(`.o_ChatWindowHeader`).click()
+    );
```

Comment:
Oh ok I get it now, you added the click here.
Then yes it's fine to remove the click and remove the `verifyStep`, and also probably remove the `mockRPC` itself.

Reply:
@seb-odoo Test cases Added.